code/game/objects/items/bouquets.dm

```diff
@@ -0,0 +1,16 @@
+// Bouquets
+/obj/item/bouquets/bouquet
```

Comment:
`/obj/item/bouquets/bouquet`  is a weird typepath, especially since item/bouquets in of itself doesn't exist

Reply:
This was because I thought the problem I was having with undefined path was because It needed to be bouquets/bouquet, but yeah, it isnt